Source/Core/VideoCommon/VideoCommon.h

```diff
@@ -57,6 +63,21 @@ struct TargetRectangle : public MathUtil::Rectangle<int>
 #endif
 };
 
+// This structure should only be used to represent a rectangle in EFB
+// coordinates, where the origin is at the upper left and the frame dimensions
+// are 640 x 528.
+using EFBRectangle = TaggedRectangle<struct EFBRectangleTag>;
```

Comment:
Hmm that doesn't seem like a comparable situation to me. The iterator tags are meant to be referenced in other code, where these tags should never be used aside from the single typedef.